### ISSUE TITLE
regression: ec2 module forgets about encrypted volumes

### ISSUE THREAD

**Issue by danvaida** [ansible/ansible-modules-core#1629]
##### Issue Type:

Bug Report
##### Ansible Version:

```
$ ansible --version
ansible 1.9.2 (stable-1.9 db5ea10ed2) last updated 2015/06/29 13:18:39 (GMT +200)
  lib/ansible/modules/core: (detached HEAD 50912c9092) last updated 2015/06/29 13:17:32 (GMT +200)
  lib/ansible/modules/extras: (detached HEAD dec7d95d51) last updated 2015/06/29 13:17:32 (GMT +200)
  v2/ansible/modules/core: (detached HEAD 85c8a892c8) last updated 2015/05/14 12:51:48 (GMT +200)
  v2/ansible/modules/extras: (detached HEAD 70ea058563) last updated 2015/05/14 12:51:51 (GMT +200)
  configured module search path = None
```
##### Ansible Configuration:

n/a
##### Environment:

FC21
##### Summary:

Until I upgraded to the stated branch, my play was working fine. Unfortunately, I don't remember the previous version. Anyway, here goes: I am trying to spin up an ec2 instance and use a volume snapshot as a dev. Nothing to do with encryption. I tried specifying _encryption: false_ in the volumes block but it didn't help. Also, taking a look at the ec2.py code, there is nothing about encryption...
##### Steps To Reproduce:

```
- name: spinning up ec2
  local_action:
    module: ec2 
    exact_count: 1
    count_tag: "{{ count_tag }}"
    region: "{{ region }}"
    zone: "{{ zone1a }}"
    instance_type: "{{ instance_type }}"
    image: "{{ ami }}"
    group_id:
      - "{{ sg_creation.group_id }}"
    vpc_subnet_id: "{{ vpc.subnets[0].id }}"
    key_name: "{{ keypair }}"
    assign_public_ip: yes
    wait: yes
    wait_timeout: 300
    volumes:
    - device_name: /dev/xvda
      volume_size: 25
      device_type: gp2
      delete_on_termination: true
    - device_name: /dev/xvdb
      snapshot: snap-xxx
      device_type: gp2
      delete_on_termination: true
    instance_tags: "{{ count_tag }}"
  register: ec2
```
##### Expected Results:

The instance is created with an attached EBS volume from the indicated snapshot.
##### Actual Results:

```
TASK: [ec2 | spinning up ec2] **************************** 
failed: [localhost -> 127.0.0.1] => {"failed": true}
msg: Instance creation failed => InvalidParameter: Parameter encrypted is invalid. You cannot specify the encrypted flag if specifying a snapshot id in a block device mapping.

FATAL: all hosts have already failed -- aborting
```

**Comment by danvaida** [comment]
Well, it seems like it doesn't have anything to do with my specific volumes block. Even this block renders the same behavior:

```
     - device_name: /dev/xvda
       volume_size: 8
       device_type: gp2
```

**Comment by bennojoy** [comment]
@danvaida  this has been fixed latest devel  via #1534

**Comment by danvaida** [comment]
Oh yeah, works with ea913bada2
Forgot to do the recursive submodules update :)
Thanks!
